docs/CHANGELOG.rst

```diff
@@ -59,6 +59,7 @@ Fixed
 - Set collection name to avoid warnings in test output
 - Improve Python 3 compatibility
 - Fix setting descriptor schema on create
+- Fixed compatibility of ``assertJSON`` with Python3
```

Comment:
This is redundant, as it is already covered two lines above ("Improve Python 3 compatibility").